config/locales/user_mailer/es.yml

```
@@ -41,8 +40,7 @@ es:
         horas, recibirá un correo electrónico que le pedirá que confirme la
         eliminación de su cuenta %{app}. Su cuenta no se eliminará hasta que
         confirme.'
-      subject: Cómo eliminar su cuenta de login.gov
-    account_verified:
+      subject: Cómo eliminar su cuenta de Login.gov account_verified:
```

Some line breaks eaten similar to the identity-site PR:

```suggestion
      subject: Cómo eliminar su cuenta de Login.gov
    account_verified:
```